src/main/webapp/WEB-INF/jsp/history/unmoderated.jsp

```diff
@@ -27,7 +27,7 @@
 			<div class="stats">
 				<c:set var="information" value="${entry.value[fn:length(entry.value)-1]}"/>
 				<span class="last-updated-at"><tags:prettyTime time="${information.createdAt}" /></span>
-				<span>- �ltimo usu�rio a editar :</span>
+				<span>- �ltimo usu�rio a editar :</span>
```

Comment:
And this :)
Btw, I guess your editor is not using utf-8 as the default encoding, can you change it?